lib/simctl.js

```diff
@@ -1,12 +1,11 @@
 import Q from 'q';
 import cp from 'child_process';
-import { sleep, retry, nodeifyAll } from 'asyncbox';
+import { sleep, retry } from 'asyncbox';
 import npmlog from 'npmlog';
-import { mapify } from 'es6-mapify';
 
 let log = global._global_npmlog || npmlog;
 
-function simExec (cmd:string, timeout:number, args:Array = []) {
+function simExec (cmd/*:string*/, timeout/*:number*/, args/*:Array*/ = []) {
```

Comment:
Actually been playing with this, it's pretty cool, appium-gulp-plugins now optionally support rtts-type, commented or not:

Try:
```
gulp once --rttsAssert
gulp watch --rttsAssert
```

wrote a test https://github.com/sebv/node-simctl/blob/better-package/test/simctl-specs.js#L61